db/migrate/20161229080315_add_comments_count_to_events.rb

```diff
@@ -0,0 +1,18 @@
+class AddCommentsCountToEvents < ActiveRecord::Migration
+  class TempComment < ActiveRecord::Base
+    self.table_name = 'comments'
+  end
+
+  class TempEvent < ActiveRecord::Base
+    self.table_name = 'events'
+  end
+
+  def change
+    add_column :events, :comments_count, :integer, default: 0, null: false
+
+    TempEvent.all.each do |event|
+      comments_count = Comment.where(commentable_id: event.id, commentable_type: 'Event').count
```

Comment:
If you used just `Event` you could do `event.comments` instead of that entire `where`-clause.